pages/linux/fastfetch.md

```diff
@@ -0,0 +1,29 @@
+# fastfetch
+
+> A neofetch-like tool but, faster & prettier.
+> Modify ~/.config/fastfetch/config.conf for persistent flags.
+> More information: <https://github.com/LinusDierheimer/fastfetch>.
+
+- Display system information as configured in ~/.config/fastfetch/config.conf:
+
+`fastfetch`
+
+- Set the structure of the fetch:
+
+`fastfetch --structure <structure>`
+
+- Load a custom config file:
+
+`fastfetch --load-config <file>`
+
+- Set the logo to use:
+
+`fastfetch --logo`
+
+- Set the color of the keys:
+
+`fastfetch --color`
```

Comment:
whats the syntax here? is it hex or just words such as "blue"

Reply:
Yup, it accepts the strings with the color names.